packages/core/src/App/Containers/WarningScamMessageModal/correct-message-content.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+const CorrectMessageContent = () => (
+    <div className='correct-link__container'>
+        <Icon icon='IcAccountTick' className='warning-scam-message__icon' size={14} />
+        <div className='correct-link__message-container'>
+            <Text>
+                <Localize i18n_default_text='Only log in to your account at this secure link, never elsewhere.' />
+            </Text>
+            <div className='correct-link__link-container'>
+                <Text color={'var(--text-general)'}>https://app.deriv.com</Text>
```

Comment:
- Look into Text.jsx color props, default is already `general`
- reuse `website_domain` instead of `https://app.deriv.com`
```suggestion
                <Text>https://app.deriv.com</Text>
```